n_advisor/calculate.py

```diff
@@ -51,10 +51,9 @@ def cal(rot, fpr, cpr, dis, fer):
         MaxN = df.iloc[i]["MaxN"]
 
         for j in range(len(xn)):
-            if lx == "QP" or lx == "Q":  # quandratic-plateau
-                if B / (-2 * A) < MaxN:
-                    x0 = B / (-2 * A)
-                else:
+            if lx == "QP":  # quandratic-plateau
+		x0=B / (-2 * A)
```

Comment:
The indention here looks incorrect. Is this line supposed to be part of `if lx == "QP"`: block?